alpha/lib/model/MediaServerNode.php

```diff
@@ -109,4 +112,14 @@ public static function modifyUrlForVodFromLive($liveUrl, DeliveryProfileDynamicA
 		return $liveUrl;
 	}
 
+	public static function getExplicitLiveUrl($liveUrl, LiveStreamEntry $entry)
+	{
+		$userType = static::USER_TYPE_ADMIN;
+		if ($entry->getExplicitLive() && !$entry->canViewExplicitLive())
```

Comment:
In the original code if $entry->getExplicitLive() was not set, then nothing was added to the URL

Reply:
this behaviour is kept -  it protected from WowzaMediaServerNode [line 264](https://github.com/kaltura/server/pull/10599/files#diff-5ef80af7d044d33fdd433e5eb4257027948c3b601fb25b8959d6c2f6cc03320cR264) -
for the other implementation of "MediaServerNode" (which is only the LiveNG serverNode) we want to add "/tl/.../" to the url regardless of "$entry->getExplicitLive()"